test.c

```diff
@@ -3313,9 +3313,9 @@ test_message_count_body (const struct message *message)
 }
 
 void
-test_simple (const char *buf, enum http_errno err_expected)
+test_simple_type (const char *buf, enum http_errno err_expected, enum http_parser_type type)
```

Comment:
It is over 80 yes, but there are also many other lines in the file that are over 80 so I assumed that wasn't being observed, at least for that file...